etc/profile-a-l/kontact.profile

```diff
@@ -0,0 +1,72 @@
+# Firejail profile for kontact
```

Comment:
Firecfg?

Reply:
I'll add that, thanks for reminding me, added to my system but forgot to do so here.